relayer-cli/src/commands/query/client.rs

```diff
@@ -19,6 +20,60 @@ use crate::conclude::Output;
 use crate::error::{Error, Kind};
 use crate::prelude::*;
 
+/// Query clients command
+#[derive(Clone, Command, Debug, Options)]
+pub struct QueryAllClientsCmd {
+    #[options(free, help = "identifier of the chain to query")]
+    chain_id: Option<ChainId>,
```

Comment:
this param should be mandatory
```suggestion
    chain_id: ChainId,
```